napari_aicssegmentation/view/workflow_steps_view.py

```diff
@@ -38,8 +38,8 @@ def __init__(self, controller: IWorkflowStepsController):
 
         # TODO: replace this with connection to model (first page selection)
         engine = WorkflowEngine()
-        self.workflow = engine.workflow_definitions[3]
-        self.all_steps = self.workflow.steps
+        self.workflow = engine.get_executable_workflow(selected_workflow, controller.model.selected_layer.data)
```

Comment:
These type of operations should be happening in the controller, not in the view. The controller already has an instance of the WorkflowEngine to work with. 
To stay consistent, the proper workflow would be: 

1. Controller get the executable workflow from engine
2. Controller puts workflow into the model (or passes it to the view in some other way)
3. View creates widget using workflow from model

Cleanest way I can see this happening would be to change `SegmenterModel.active_workflow` from a string into a `Workflow` object. When the user selects the workflow, just load the executable workflow into the model instead of just the name right here -> https://github.com/AllenCell/napari-aicssegmentation/blob/20cee3e2e9b08d545b6e9d32d9d64d2435bbc6ea/napari_aicssegmentation/controller/workflow_select_controller.py#L68